Shop/src/Computer.java

```diff
@@ -0,0 +1,16 @@
+public class Computer extends ElectronicsProduct {
+
+    public Computer(String name, double price, int quantity, String ageRestriction, int guaranteePeriod) {
```

Comment:
ai putea scoate guaranteePeriod din constructor si sa pui direct valoarea, avand in vedere ca exista o singura valoare acceptata (24). Acelasi lucru este valabil si pentru Appliance doar ca trebuie setata valoarea 6.